--- origsrc/jbig2dec-0.11/configure.ac	2010-02-02 12:55:40.000000000 -0600
+++ src/jbig2dec-0.11/configure.ac	2010-08-12 23:00:54.521300600 -0500
@@ -45,7 +45,7 @@ if test "x$ac_cv_want_libpng" != "xno";
   fi
   dnl libpng requires pow() which may be in libm
   AC_SEARCH_LIBS([pow], [m])
-  AC_CHECK_LIB([png], [png_check_sig], [
+  AC_CHECK_LIB([png], [png_sig_cmp], [
     AC_CHECK_LIB([z], [deflate], [
       AC_DEFINE(HAVE_LIBPNG, 1, [Define if libpng is available (-lpng)])
       LIBS="-lpng -lz $LIBS"
