--- origsrc/jbig2dec-0.11/Makefile.am	2009-07-15 23:44:49.000000000 -0500
+++ src/jbig2dec-0.11/Makefile.am	2010-08-12 23:01:28.330234300 -0500
@@ -27,7 +27,7 @@ noinst_PROGRAMS = test_sha1 test_huffman
 jbig2dec_SOURCES = jbig2dec.c sha1.c sha1.h \
 	jbig2.h jbig2_image.h getopt.h \
 	os_types.h config_types.h config_win32.h
-jbig2dec_LDADD = libjbig2dec.la @LIBOBJS@
+jbig2dec_LDADD = libjbig2dec.la @LIBOBJS@ $(PNG_LIBS)
 
 dist_man_MANS = jbig2dec.1
 
--- origsrc/jbig2dec-0.11/configure.ac	2010-02-02 12:55:40.000000000 -0600
+++ src/jbig2dec-0.11/configure.ac	2010-08-12 23:00:54.521300600 -0500
@@ -48,7 +48,7 @@ if test "x$ac_cv_want_libpng" != "xno";
   AC_CHECK_LIB([png], [png_check_sig], [
     AC_CHECK_LIB([z], [deflate], [
       AC_DEFINE(HAVE_LIBPNG, 1, [Define if libpng is available (-lpng)])
-      LIBS="-lpng -lz $LIBS"
+      PNG_LIBS="-lpng -lz"
       AC_LIBOBJ([jbig2_image_png])
       have_libpng="yes"
     ])
@@ -59,6 +59,7 @@ if test "x$have_libpng" != "xyes"; then
   CFLAGS="$save_cflags"
   LDFLAGS="$save_ldflags"
 fi
+AC_SUBST(PNG_LIBS)
 
 # Checks for header files.
 AC_HEADER_STDC
